handbook/marketing/index.md

```diff
@@ -32,9 +58,7 @@
     - [Naming Process for Products, Features and Programs](brand/naming_process_for_products_features_and_programs.md)
     - [Working with Us](brand/production_process.md)
 
-- Marketing requests and feedback
-    - [Make a request or share an idea](https://form.asana.com/?k=hNaq42PNshdQ1FjjEUKjLA&d=7195383522959)
-    - [Sign up for Marketing office hours](https://docs.google.com/spreadsheets/d/1o8rHntP0j6-JFryhc0_g6MKEI6l3dZE7NTKasyQx46M/edit#gid=0)
+
 - [Marketing roles - open positions](https://sourcegraph.com/careers)
 
 ## Members
```

Comment:
@rebeccadee is there a magical way to have this section auto-update when we update it on the "Marketing Team" page? Or do we need to always update in two places?

Reply:
@jennybergen I'm not sure I understand your question – this _is_ the marketing team page?